docs/proposals/pepxxx_wheel_variant_support.md

```diff
@@ -669,88 +721,47 @@ Examples:
-to group features defined by a single provider, and avoid conflicts should multiple providers define a feature with the
-same name. These keys are restricted to lowercase letters, digits and underscores, to make it easier to preserve
-consistency between different providers. In particular, uppercase characters are disallowed to avoid different spellings
-of the same name. The character set for values is more relaxed, to permit values resembling versions.
-
-Variant features can be declared as allowing multiple values. If that is the case, these values are matched as a logical
-disjunction, i.e. only a single value needs to be supported. Features are treated conjunctively, i.e. all of them need
-to be supported. This provides some flexibility in designating variant compatibility while avoiding having to implement
-a complete boolean logic.
+```
+{namespace} :: {feature_name} :: {feature_value}
+```
 
-**This hierarchical structure enables:**
+The namespace must consist only of `0-9`, `a-z` and `_` ASCII characters (`^[a-z0-9_]+$`).
+It must correspond to a single variant provider.
 
-- Organized property management without naming conflicts
-- Independent development of provider plugins
-- Extensible support for new hardware and software capabilities without requiring changes to tools or a new PEP
-- Clear ownership and validation responsibilities
+The feature name must consist only of `0-9`, `a-z` and `_` ASCII characters (`^[a-z0-9_]+$`).
+It must correspond to a valid feature name defined by the respective variant provider in the namespace.
 
-#### Variant property format
+The feature value must consist only of `0-9`, `a-z`, `_` and `.` ASCII Characters (`^[a-z0-9_.]+$`).
+It must correspond to a valid value defined by the respective variant provider for the feature.
 
-Variant properties use a structured 3-tuple format inspired by [PEP 301 for Trove Classifiers](https://peps.python.org/pep-0301/#distutils-trove-classification)
+If a feature is marked as "multi-value" by the provider, a single variant wheel may define multiple properties
+sharing the same namespace and feature name. Otherwise, only a single value can correspond to a single namespace
+and feature name within a variant wheel.
 
-```
-namespace :: feature-name :: feature-value
-```
+For a variant wheel to be considered compatible with the system, all of the features defined within it must be
+determined to be compatible. For a feature to be compatible, at least a single value corresponding to it must be
+compatible.
 
 Examples:
 
 ```
-nvidia :: cuda_version_lower_bound :: 12.8
+# all of the following must be supported
 x86_64 :: level :: v3
-aarch64 :: version :: 8.1a
 x86_64 :: avx512_bf16 :: on
+nvidia :: cuda_version_lower_bound :: 12.8
+# additionally, at least one of the following must be supported
+nvidia :: sm_arch :: 120_real
+nvidia :: sm_arch :: 110_real
```

Comment:
Changed the example from "random properties that could be defined" to "a set of possible properties in a single wheel", so I could also explain the multi-value behavior.